src/about.md

```diff
@@ -140,14 +138,13 @@ fn run() -> Result<()> {
     Ok(())
 }
 
-quick_main!(run);
 ```
 
 This is using the `error_chain!` macro to define a custom `Error` and
 `Result` type, along with automatic conversions from two standard
 library error types. The automatic conversions make the `?` operator
-work. The `quick_main!` macro generates the actual `main` function and
-prints out the error if one occurred.
+work. Since rust 1.26, the main function can return a value, no need 
+to use a custom `run` function like before.
```

Comment:
This sentence 
`Since rust 1.26, the main function can return a value, no need 
to use a custom `run` function like before.` should not be necessary

Reply:
Thanks for the reviews, i will do this asap